Zend/zend_compile.c

```diff
@@ -3862,6 +3863,24 @@ int zend_compile_func_get_args(znode *result, zend_ast_list *args) /* {{{ */
 }
 /* }}} */
 
+int zend_compile_func_array_key_exists(znode *result, zend_ast_list *args) /* {{{ */
+{
+	znode subject, needle;
+
+	if (args->children != 2
+	 || args->child[0]->kind == ZEND_AST_UNPACK
```

Comment:
Why is it useless? We're checking against UNPACK in all the function specializations (though it would probably make more sense to only do this check once).

Reply:
We may check against UNPACK only the last argument.

Reply:
I don't think that's enough, because we're not going through compile_args, which is where we would usually enforce that unpack is only used in the last argument.

Reply:
@nikic Just noticed 850b53df0e3aa1a898db000244290fd0b7ecc8b6, so dropping the condition here, keeping only the `children != 2`. :+1: